### PR TITLE
Reused common default docker parm.

### PR DESCRIPTION
**Pull request by yanggangtony**
Signed-off-by: yanggang <gang.yang@daocloud.io>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Reused ```common/constants``` define parm..

```release-note
None
```

### CODE REVIEW

File: keadm/cmd/keadm/app/cmd/debug/check.go [hunk 1]
```diff
@@ -18,6 +18,8 @@ import (
 
 	"github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/common"
 	"github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/util"
+
+	"github.com/kubeedge/kubeedge/common/constants"
```
Comment (gy95) [inline keadm/cmd/keadm/app/cmd/debug/check.go:22]: this should be moved to line21

Comment (yanggangtony) [reply keadm/cmd/keadm/app/cmd/debug/check.go]: thanks, i install the 'goimports plugin' , and let it format for me when 'ctrl+s'.
Had updated.

**Comment by yanggangtony** [comment]
still ci lint err,
```keadm/cmd/keadm/app/cmd/debug/check.go:14: File is not `gci`-ed with -local github.com/kubeedge (gci)

keadm/cmd/keadm/app/cmd/debug/check.go:20: File is not `gci`-ed with -local github.com/kubeedge (gci)
```

Is the ci not works with goimports?

File: keadm/cmd/keadm/app/cmd/debug/check.go [hunk 2]
```diff
@@ -11,13 +11,13 @@ import (
 	"strings"
 	"time"
 
+	"github.com/kubeedge/kubeedge/common/constants"
+	"github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/common"
+	"github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/util"
```
Comment (wackxu) [inline keadm/cmd/keadm/app/cmd/debug/check.go:16]: please format the import order

Comment (yanggangtony) [reply keadm/cmd/keadm/app/cmd/debug/check.go:16]: hi, now i am use the "goimports" to save the file to keep the import order.
What is the format style is good for the ci?

Comment (yanggangtony) [reply keadm/cmd/keadm/app/cmd/debug/check.go]: @gy95 
thanks for explain .
so kubeage is used 'gofmt' to format the code , note 'goimport'.
Fix it

**Review by fisherxu — COMMENTED** [review]
Th default value for runtime in edgecore's config has not updated in #4533? @gy95

**Comment by gy95** [comment]
> The default value for runtime in edgecore's config has not updated in #4533? @gy95

keadm debug related commands not changed

**Comment by fisherxu** [comment]
> > The default value for runtime in edgecore's config has not updated in #4533? @gy95
> 
> keadm debug related commands not changed

ok. 
/lgtm
/approve

**Pull request merged**